metrics/repl.lag/lag.go

```diff
@@ -54,28 +68,29 @@ func (c *Lag) Help() blip.CollectorHelp {
 		Domain:      DOMAIN,
 		Description: "Replication lag",
 		Options: map[string]blip.CollectorHelpOption{
-			OPT_WRITER: {
-				Name:    OPT_WRITER,
-				Desc:    "Type of heartbeat writer",
-				Default: "blip",
+			OPT_LAG_SOURCE: {
```

Comment:
Let's not change the option because it's backwards-incompatible. `writer = blip|pfs` works because Perf Schema is the writer/where heartbeats are written.

Reply:
I have also prefixed the heartbeat related options with `heartbeat-` which are not backward compatible as well. Maybe I should revert them too? The options are:

```
source-id
source-role
table
```

I prefixed them because they have no relevance when the using PFS.

Reply:
I will revert them other changes as well, though they are bit ambiguous without context of heartbeat writer, documentation should fill that context.